client/db/src/sql/mod.rs

```diff
@@ -820,6 +820,15 @@ impl<Block: BlockT<Hash = H256>> fc_api::Backend<Block> for Backend<Block> {
 		self
 	}
 
+	async fn first_block_hash(&self) -> Result<Block::Hash, String> {
+		// Retrieves the block hash for the latest indexed block, maybe it's not canon.
```

Comment:
Please update the comment here; it's for the earliest indexed block instead of the latest.